build-scripts/bundle.cjs

```diff
@@ -125,7 +125,6 @@ module.exports.babelOptions = ({ latestBuild, isProdBuild, isTestBuild }) => ({
             { name: "svg", encapsulation: "svg" },
             { name: "css", encapsulation: "style" },
           ],
-          "@polymer/polymer/lib/utils/html-tag": ["html"],
```

Comment:
Don't remove this yet.  It also applies to dependencies, so this would also stop minifying `paper-*`, which is probably why the bundle size went slightly up instead of down.

Reply:
Reverted in #18795

Reply:
Thanks for fixing. I wasn't aware it also minified dependencies.